docs/source/history.rst

```diff
@@ -5,6 +5,44 @@ Release history
 
 .. towncrier release notes start
 
+pytest-trio 0.8.0 (2022-11-01)
+------------------------------
+
+Features
+~~~~~~~~
+
+- If a test raises an ``ExceptionGroup`` (or nested ``ExceptionGroup``\ s) with only
```

Comment:
Could probably make ExceptionGroup a link? (intersphinx to the Python docs)

Reply:
Sorry, our release process mentions that no review is needed, and while this is a valid improvement, I uploaded the release to PyPI already. Would be happy to accept a PR from anyone that fixes that, even though it won't warrant a new release.